arlas-persistence-tests/src/test/java/io/arlas/persistence/rest/PersistenceIT.java

```diff
@@ -410,6 +410,35 @@ public void test19PostData() {
         createData(technical, "myNewDocument", Collections.singletonList("group/private"),Collections.EMPTY_LIST)
                 .then().statusCode(403);
     }
+    @Test
+    public void test20ExistsNot() {
+        givenForUser(technical)
+                .pathParam("zone", dataZone)
+                .pathParam("key", "foo")
+                .when()
+                .get(arlasAppPath.concat("resource/exists/{zone}/{key}"))
+                .then().statusCode(200)
+                .contentType(ContentType.JSON)
+                .body("exists", equalTo(false));
+    }
+
+    @Test
+    public void test21ExistsById() {
```

Comment:
`ById` but you don't use the id to retrieve the data

Reply:
Indeed. Fixed!